manifests/m/Musescore/Musescore/4.2.0/Musescore.Musescore.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Musescore.Musescore
+PackageVersion: 4.2.0

```

```suggestion
PackageVersion: 4.2.0.233521124
```
Should match control panel. The folder name will need to be updated after making this change.